packages/jest/test/__snapshots__/react-enzyme.test.js.snap

```diff
@@ -446,11 +446,13 @@ exports[`enzyme shallow displayName 1`] = `
 `;
 
 exports[`enzyme shallow empty styled 1`] = `
-<div
-  className="emotion-0 emotion-1"
->
-  Hello
-</div>
+<Fragment>
```

Comment:
I wonder if this shouldn't render `<>` but I'm not sure at the moment if this is under our control, gonna check this out

Reply:
I've fixed the serialization tests here - but I have a feeling that our test suite is missing some cases that should currently fail. I will do some further investigation soon.